README.md

```diff
@@ -12,12 +13,16 @@ client has permission to access resources on behalf of the resource owner.
 
 OAuth2 provides several different methods for the client to obtain
 authorization. At the time of writing, this library only supports the
-[AuthorizationCodeGrant][] method, but further methods may be added in the
-future. The following example uses this method to authenticate, and assumes
-that the library is being used by a server-side application.
+[AuthorizationCodeGrant][] and [resourceOwnerPasswordGrant][] methods, but
+further methods may be added in the future. The following example uses this
+method to authenticate, and assumes that the library is being used by a server-side
+application.
 
-[AuthorizationCodeGrant]: https://api.dartlang.org/apidocs/channels/stable/#oauth2/oauth2.AuthorizationCodeGrant
+[AuthorizationCodeGrant]: http://www.dartdocs.org/apidocs/channels/stable/#oauth2/oauth2.AuthorizationCodeGrant
+[resourceOwnerPasswordGrant]: http://www.dartdocs.org/apidocs/channels/stable/#oauth2/oauth2.resourceOwnerPasswordGrant
```

Comment:
It scans the pub packages. You can always use `latest` in place of a version number to link to the latest version of the package docs (e.g. `http://www.dartdocs.org/documentation/oauth2/latest/index.html#...`). Note that it *doesn't* use the same directory structure as the old `api.dartlang.org` docs.